docs/cilium.md

```diff
@@ -0,0 +1,7 @@
+# Azure CNI Powered By Cilium
+
+Azure CNI powered by [Cilium](https://cilium.io/) provides native support for the next generation Cilium eBPF dataplane in AKS clusters running Azure CNI. It offers Pod networking, basic [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/), and high-performance service load balancing. The eBPF dataplane is available in both [VNet mode](https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni#dynamic-allocation-of-ips-and-enhanced-subnet-support) and [Overlay mode](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay) of Azure CNI.
```

Comment:
can it be `highly scalable k8s network policies and high performance service load balancing with eBPF`

Reply:
done